Resources/config/consumers.yml

```diff
@@ -9,9 +8,8 @@ services:
           - [ setId, ['smartesb.consumers.queue']]
           - [ setSmartesbHelper, ['@smartesb.helper']]
           - [ setEventDispatcher, ['@event_dispatcher']]
-  smartesb.consumers.async_queue:
-      class: "%smartesb.consumers.async_queue.class%"
-      arguments: ['@smartesb.amqp.queue_manager']
+  smartesb.async_consumers.queue:
```

Comment:
We should rename this service to something that start with `smartesb.consumers.{name of the consumer}`. The reason of that is in Frameworkbundle the class Tools/Helper/SmartesbHelper.php allows to get consumers based on consumer name, but expects that the beginning of the service id is `smartesb.consumers` and right now this service can't not be get using that helper

Once this is changed, a change will be needed in FrameworkExtension to use the new service id

Reply:
Done.